### PR TITLE
CLDR-13140 javax.xml.bind.DatatypeConverter -> java.util.Base64

### PR DESCRIPTION
**Pull request by srl295**
- [CLDR-13140] 
- follow on to [CLDR-5507] in f5262780b999a48220e96639bcee6a0e1ec3a47a !

fyi @hagbard 

[CLDR-13140]: https://unicode-org.atlassian.net/browse/CLDR-13140
[CLDR-5507]: https://unicode-org.atlassian.net/browse/CLDR-5507

### CODE REVIEW

File: tools/cldr-apps/src/org/unicode/cldr/web/CookieSession.java [hunk 1]
```diff
@@ -22,10 +23,7 @@
 import java.util.Set;
 import java.util.TreeSet;
 
-import javax.xml.bind.DatatypeConverter;
-
 import org.unicode.cldr.util.CLDRConfig;
-import org.unicode.cldr.util.Level;
```
Comment (hagbard) [inline tools/cldr-apps/src/org/unicode/cldr/web/CookieSession.java]: any reason for removing this import?

File: tools/cldr-apps/src/org/unicode/cldr/web/CookieSession.java [hunk 2]
```diff
@@ -927,7 +884,7 @@ public String getOrgCoverageLevel(String locale) {
             level = WebContext.COVLEV_DEFAULT_RECOMMENDED_STRING;
         } else {
             org.unicode.cldr.util.Level l = StandardCodes.make().getLocaleCoverageLevel(myOrg, locale);
```
Comment (hagbard) [inline tools/cldr-apps/src/org/unicode/cldr/web/CookieSession.java]: I know it's not part of this review, but damn that looks like a '1' in my font. The snippet "(l == org.unicode.cldr.util.Level.UNDETERMINED)" is particularly easy to misread.
Perhaps change it to "coverageLevel" ?

File: tools/cldr-apps/src/org/unicode/cldr/web/CookieSession.java (hunk 1 above)
Comment (srl295) [reply tools/cldr-apps/src/org/unicode/cldr/web/CookieSession.java]: Originally there were multiple classes named `Level`, this disambiguates.

File: tools/cldr-apps/src/org/unicode/cldr/web/CookieSession.java (hunk 2 above)
Comment (srl295) [reply tools/cldr-apps/src/org/unicode/cldr/web/CookieSession.java]: glad to ob1ige

File: tools/cldr-apps/src/org/unicode/cldr/web/CookieSession.java (hunk 1 above)
Comment (srl295) [reply tools/cldr-apps/src/org/unicode/cldr/web/CookieSession.java]: VoteResolver.level vs cldr.util.Level

Comment (srl295) [reply tools/cldr-apps/src/org/unicode/cldr/web/CookieSession.java]: @hagbard reinstated Level, PTAL

**Pull request merged**